kernel/src/hil/screen.rs

```diff
@@ -135,6 +186,15 @@ pub trait Screen {
     /// - `EBUSY`: Another write is in progress.
     fn write(&self, buffer: &'static mut [u8], len: usize) -> ReturnCode;
 
+    /// Continues the previous previous write command
+    /// When finished, the driver will call the `write_complete()` callback.
+    ///
+    /// Return values:
+    /// - `SUCCESS`: Write is valid and will be sent to the screen.
+    /// - `EINVAL`: Write is invalid or length is wrong.
+    /// - `EBUSY`: Another write is in progress.
+    fn write_continue(&self, buffer: &'static mut [u8], len: usize) -> ReturnCode;
```

Comment:
I am not clear on why this function is being added to the trait. What can lead to a write beginning, but not being finished? How should clients know when `write_continue()` needs to be called?

Reply:
The frame buffer data might be larger than the allocated RAM, for instance when performing a fill. 

- `write` sets the memory address at the beginning of the screen frame window
- `write_continue` does the same thing as `write`, but does not reset the frame windows position

Reply:
That makes sense. Can you add that description to the comments in this file? I initially misread this as `write_continue()` somehow resuming a paused write.

Reply:
Done